external/zillow/tests.py

```diff
@@ -21,4 +25,13 @@ def test_get_zillow_housing(self):
             address="Jay St", zipcode="11201", show_rent_z_estimate=True
         )
         for housing in results:
-            self.assertTrue(isinstance(housing, ZillowHousing))
+            self.assertTrue(isinstance(housing, ZillowHousingResponse))
+
+    @mock.patch("external.zillow.fetch.get_zws_id", get_zws_id_stub)
+    @mock.patch("external.zillow.fetch.requests")
+    def test_get_zillow_housing_no_match_results(self, mock_requests):
+        mock_requests.get().content = get_zillow_error_response()
+        results = get_zillow_housing(
+            address="Jay St", zipcode="11201", show_rent_z_estimate=True
+        )
+        self.assertEqual(len(results), 0)
```

Comment:
It would be nice to add comments about why should the result length is 0.

Reply:
Added a comment